agent/engine/dockerstate/json.go

```diff
@@ -24,21 +24,23 @@ import (
 // These bits of information should be enough to reconstruct the entire
 // DockerTaskEngine state
 type savedState struct {
-	Tasks         []*api.Task
-	IdToContainer map[string]*api.DockerContainer `json:"IdToContainer"` // DockerId -> api.DockerContainer
-	IdToTask      map[string]string               `json:"IdToTask"`      // DockerId -> taskarn
-	ImageStates   []*image.ImageState
+	Tasks          []*api.Task
+	IdToContainer  map[string]*api.DockerContainer `json:"IdToContainer"` // DockerId -> api.DockerContainer
+	IdToTask       map[string]string               `json:"IdToTask"`      // DockerId -> taskarn
+	ImageStates    []*image.ImageState
+	ENIAttachments []*api.ENIAttachment `json:enis`
```

Comment:
Please update the documentation in [state_manager.go](https://github.com/aws/amazon-ecs-agent/blob/enis/agent/statemanager/state_manager.go#L50) that a new field has been added, with a description of the same.